ClimsoftVer4/ClimsoftVer4/formAWSRealTime.vb

```diff
@@ -1,4 +1,20 @@
-﻿Imports System.IO
+﻿'CLIMSOFT - Climate Database Management System
```

Comment:
A minor point on coding style:  We would usually have a space between the comment character (') and the first word on the line, e.g.
    `'CLIMSOFT`
would be replaced with
    `' CLIMSOFT`